examples/pxe_infra.py

```diff
@@ -0,0 +1,161 @@
+from pyinfra import host
```

Comment:
I think this would be improved if we put the PXE example in it's own folder, `examples/pxe/` containing a `Vagrantfile` and this file with a README containing the instructions. Should make it much easier to give it a try w/o editing any files.

Reply:
Good idea. Will do.